Deploy/bash/deploy-data-module.sh

```diff
@@ -0,0 +1,345 @@
+#!/bin/bash
+
+set -e #Exit on first error
+set -x #Be verbose
+
+#
+# Change to working directory (if not set by a parent script)
+#
+
+if [ -z "${START_DIR}" ]; then
+  START_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+  cd "${START_DIR}"
+fi
+
+#
+# Check vars are not empty before proceeding
+#
+
+echo "Check vars are not empty before proceeding..."
+if [ -z "${AWS_ACCOUNT_NUMBER_PARAM}" ] \
+    || [ -z "${CLOUD_TAMER_PARAM}" ] \
+    || [ -z "${CMS_ENV_PARAM}" ] \
+    || [ -z "${CPM_BACKUP_DB_PARAM}" ] \
+    || [ -z "${DATABASE_SECRET_DATETIME_PARAM}" ] \
+    || [ -z "${DB_ALLOCATED_STORAGE_SIZE_PARAM}" ] \
+    || [ -z "${DB_BACKUP_RETENTION_PERIOD_PARAM}" ] \
+    || [ -z "${DB_BACKUP_WINDOW_PARAM}" ] \
+    || [ -z "${DB_COPY_TAGS_TO_SNAPSHOT_PARAM}" ] \
+    || [ -z "${DB_INSTANCE_CLASS_PARAM}" ] \
+    || [ -z "${DB_IOPS_PARAM}" ] \
+    || [ -z "${DB_MAINTENANCE_WINDOW_PARAM}" ] \
+    || [ -z "${DB_MULTI_AZ_PARAM}" ] \
+    || [ -z "${DB_NAME_PARAM}" ] \
+    || [ -z "${DB_SNAPSHOT_ID_PARAM}" ] \
+    || [ -z "${DEBUG_LEVEL_PARAM}" ] \
+    || [ -z "${JENKINS_AGENT_SEC_GROUP_ID_PARAM}" ] \
+    || [ -z "${PARENT_ENV_PARAM}" ] \
+    || [ -z "${POSTGRES_ENGINE_VERSION_PARAM}" ] \
+    || [ -z "${SSH_USERNAME_PARAM}" ]; then
+  echo "ERROR: All parameters must be set."
+  exit 1
+fi
+
+#
+# Set variables
+#
+
+AWS_ACCOUNT_NUMBER="${AWS_ACCOUNT_NUMBER_PARAM}"
+
+CMS_ENV="${CMS_ENV_PARAM}"
+
+CPM_BACKUP_DB="${CPM_BACKUP_DB_PARAM}"
+
+DATABASE_SECRET_DATETIME="${DATABASE_SECRET_DATETIME_PARAM}"
+
+DB_ALLOCATED_STORAGE_SIZE="${DB_ALLOCATED_STORAGE_SIZE_PARAM}"
+
+DB_BACKUP_RETENTION_PERIOD="${DB_BACKUP_RETENTION_PERIOD_PARAM}"
+
+DB_BACKUP_WINDOW="${DB_BACKUP_WINDOW_PARAM}"
+
+DB_COPY_TAGS_TO_SNAPSHOT="${DB_COPY_TAGS_TO_SNAPSHOT_PARAM}"
+
+DB_INSTANCE_CLASS="${DB_INSTANCE_CLASS_PARAM}"
+
+DB_IOPS="${DB_IOPS_PARAM}"
+
+DB_MAINTENANCE_WINDOW="${DB_MAINTENANCE_WINDOW_PARAM}"
+
+DB_MULTI_AZ="${DB_MULTI_AZ_PARAM}"
+
+DB_NAME="${DB_NAME_PARAM}"
+
+DB_SNAPSHOT_ID="${DB_SNAPSHOT_ID_PARAM}"
+
+export DEBUG_LEVEL="${DEBUG_LEVEL_PARAM}"
```

Comment:
Exports are used for variables that are used or exclusively used by called applications like terraform executable in this case. Sometimes it will work without export, but in some cases it will not. pyCharm shell script will put warnings anywhere that it can't see where a variable is being used. This also eliminates that warning.